debian/control

```diff
@@ -22,5 +22,5 @@ Standards-Version: 4.1.2
 
 Package: docker-python-image
 Architecture: all
-Description: Saves the docker python:2.7.18-slim image to disk
+Description: Saves the docker python:2.7.18-slim and python:3.8-slim images to disk
```

Comment:
IMO, just change this to a more generic "docker python images", so it doesn't have to be updated for each new image.